src/widgets/helper/ResizingTextEdit.hpp

```diff
@@ -66,6 +66,21 @@ class ResizingTextEdit : public QTextEdit
      */
     bool completionInProgress_ = false;
 
+    /**
+     * This is set whenever the tab completion is triggered.
+     * When this variable is set, moving the cursor will not reset tab completion.
```

Comment:
I find this comment confusing, because there's `completionInProgress_` which seems to do a similar thing (from the comments' perspective). It should include that this is immediately unset and used to ignore cursor-position-change-events ("moving the cursor" suggests user-input).

Reply:
Said comment has been removed for QSignalBlocker